src/index.html

```diff
@@ -1,13 +1,297 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bang and Olufsen</title>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="images/favicon-16x16.png"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Manrope&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#" class="top-bar__logo-link">
+              <img
+              src="images/Logo-2x.png"
+              alt="Bose logo"
+              class="top-bar__logo"
+              >
```

Comment:
Need to fix code style everywhere
```suggestion
              <img
                src="images/Logo-2x.png"
                alt="Bose logo"
                class="top-bar__logo"
              >
```